src/main/webapp/js/instructor.js

```diff
@@ -152,6 +152,51 @@ function setupFsCopyModal() {
             }
         });
     });
+    
+    $('#form_copy_list').submit(fsCopyModalFormSubmitHandler);
+}
+
+/**
+ * Event handler for the copying feedback session modal's form.
+ * For submitting the form by ajax.
+ * 
+ * this should be bound to the form element
+ */
+function fsCopyModalFormSubmitHandler(e) {
+    e.preventDefault();
+    $this = $(this);
+    
+    $copyModalStatusMessage = $('#feedback-copy-modal-status');
+    
+    $.ajax({
+        type: 'POST',
+        url: $this.prop('action'),
+        data: $this.serialize(),
+        beforeSend: function() {
+            $copyModalStatusMessage.removeClass("alert alert-danger");
+            $copyModalStatusMessage.html($('<img>', {
+                                            'class':'margin-center-horizontal',
+                                            'src':'/images/ajax-loader.gif'
+                                            }
+                                         ));
+        },
+        error: function() {
+            $copyModalStatusMessage.addClass("alert alert-danger");
+            $copyModalStatusMessage.text('There was an error during submission. ' 
+                                         + 'Please close the dialog window and try again.');
+        },
+        success: function(data) {
+            if (!data.isError) {
+                if (data.redirectUrl) {
+                    window.location.href = data.redirectUrl;
+                }
```

Comment:
Is it possible for there to be no error and also no redirect URL?
What's the expected behavior in that case?